README.md

```diff
@@ -300,7 +310,22 @@ If you cannot find suitable test data on this repository, please contact us on t
       * fastq
         * 'test.fastq.gz' synthetic raw long-read sequencing reads of the genome of the mammalian-gut-residing _Bacteroides fragilis_ YCH46 bacterium (NC_006347). Originally generated for the [MAG pipeline test dataset](https://github.com/nf-core/test-datasets/tree/mag).
 
-## generic
+### pangenomics
+
+* homo_sapines
+  * pangenome
+    * 'pangenome.fa': A FASTA file which contains several related genomes for the input of `wfmash`.
+    * 'pangenome.fa.gz': A GZIP compressed FASTA file which contains several related genomes for the input of `wfmash`.
+    * 'pangenome.paf': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.
+    * 'pangenome.paf.gz': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.
+    * 'pangenome.seqwish.gfa': A GFA file which contains the pangenome graph induced by seqwish encoded in the variation graph model. It is consumed by `smoothxg`, `odgi sort`, `odgi build`, and `odgi stats`.
+    * 'pangenome.smoothxg.gfa': A GFA file which contains the `smoothxg` smoothed pangenome graph. It is consumed by `odgi sort`, and `GFAffix`.
+    * 'pangenome.gfaffix.gfa': A GFA file which was normalized with `gfaffix`. It is consumed by `vg deconstruct`, `odgi layout`, `odgi draw`, and `odgi viz`.
+  * odgi
+    * 'pangenome.og': A variation graph encoded in the binary ODGI format. It is consumed by `odgi view`.
+    * 'pangenome.lay': A binary file which holds the 2D graph layout produced by `odgi layout`. Input for `odgi draw`.
```

Comment:
```suggestion
    * 'pangenome.lay': A binary file which holds the 2D graph layout produced by `odgi layout`.
```

Reply:
This is a binary file specifically designed to be only readable by `odgi layout`. I would leave this here.